packages/flutter/lib/fix_data.yaml

```diff
@@ -11,6 +11,28 @@
 version: 1
 transforms:
 
+  # Changes made in https://github.com/flutter/flutter/pull/66043
+  - title: "Use withKind"
+    date: 2020-09-17
+    element:
+      uris: [ 'gestures.dart' ]
+      constructor: ''
+      inClass: 'VelocityTracker'
+    changes:
+      - kind: 'rename'
+        newName: 'withKind'
+      - kind: 'addParameter'
+        index: 0
+        name: 'kind'
+        style: required_positional
+        argumentValue:
+          expression: 'PointerDeviceKind.touch'
+          requiredIf: "pointerDeviceKind == ''"
+          variables:
+            pointerDeviceKind:
+              kind: 'fragment'
+              value: 'arguments[0]'
```

Comment:
I've tried this a couple different ways now. The rename works fine, but adding the parameter does not work. I can see there is an error in the file (fix_data.yaml is red-underlined in the project directory), but I don't know where the error is (https://github.com/dart-lang/sdk/issues/44890).

Reply:
Error does show up in analysis panel. So that is better, but I still can't seem to figure this fix out.